### ISSUE TITLE
Make FaultyQuoteUsage rule support multiline strings

### ISSUE THREAD

**Issue by racodond** [iwarapter/sonar-puppet#243]
No issue raised on 

```
file { 'foo':
  text => 'blabla
          \'blabla\'',  # Noncompliant
}
```

for instance

**Comment by racodond** [comment]
Fixed in https://github.com/iwarapter/sonar-puppet/pull/242

**Comment by iwarapter** [comment]
Merged.
